README.md

```diff
@@ -9,3 +9,17 @@
 - [Theo]()
 
 - Coach : [Charlie](https://github.com/Charlie-robin)
+
```

Comment:
Can you add:
- A sub title `### Name Of Job` 
  - The name of the Job you created.
- Where in the config do you need to go to update the branch?